src/controller.cpp

```diff
@@ -354,10 +356,10 @@ tf2::Transform Controller::findPositionOnPlan(
 
   const auto dist_result_ahead = distToSegmentSquared(
     current_tf2, global_plan_tf_[controller_state.current_global_plan_index],
-    global_plan_tf_[controller_state.current_global_plan_index + 1]);
+    global_plan_tf_[controller_state.current_global_plan_index + 1], estimate_pose_angle_enabled_);
   const auto dist_result_behind = distToSegmentSquared(
     current_tf2, global_plan_tf_[controller_state.current_global_plan_index - 1],
-    global_plan_tf_[controller_state.current_global_plan_index]);
+    global_plan_tf_[controller_state.current_global_plan_index], estimate_pose_angle_enabled_);
```

Comment:
You might improve a bit futher by naming `estimate_pose_angle_enabled_` to `estimate_pose_angle_` everywhere.

Reply:
@Timple fixed!